autoload/neomake/makers/ft/text.vim

```diff
@@ -1,5 +1,12 @@
 function! neomake#makers#ft#text#EnabledMakers()
-    return ['proselint']
+    return ['proselint', 'writegood']
+endfunction
+
+function! neomake#makers#ft#text#writegood()
+    return {
```

Comment:
This should use/return `neomake#makers#ft#markdown#writegood()`.
Then it's only defined in one place.